option_test.go

```diff
@@ -402,3 +403,151 @@ func TestHide(t *testing.T) {
 		require.Equal(t, "hello world", w.Body.String())
 	})
 }
+
+func TestSecurity(t *testing.T) {
+	t.Run("single security requirement", func(t *testing.T) {
+		s := fuego.NewServer()
+		basic := openapi3.SecurityRequirement{
+			"basic": []string{},
+		}
+		route := fuego.Get(s, "/test", helloWorld,
+			fuego.OptionSecurity(basic),
+		)
+
+		require.NotNil(t, route.Operation.Security)
+		require.Len(t, *route.Operation.Security, 1)
+		require.Contains(t, (*route.Operation.Security)[0], "basic")
+		require.Empty(t, (*route.Operation.Security)[0]["basic"])
+
+		r := httptest.NewRequest(http.MethodGet, "/test", nil)
+		w := httptest.NewRecorder()
+		s.Mux.ServeHTTP(w, r)
+		require.Equal(t, "hello world", w.Body.String())
+	})
+
+	t.Run("security with scopes", func(t *testing.T) {
+		s := fuego.NewServer()
+
+		route := fuego.Get(s, "/test", helloWorld,
+			fuego.OptionSecurity(
+				openapi3.SecurityRequirement{
+					"oauth2": []string{"read:users", "write:users"},
+				},
+			),
+		)
+
+		require.NotNil(t, route.Operation.Security)
+		require.Len(t, *route.Operation.Security, 1)
+		require.Equal(t,
+			[]string{"read:users", "write:users"},
+			(*route.Operation.Security)[0]["oauth2"],
+		)
+	})
+
+	t.Run("AND combination", func(t *testing.T) {
+		s := fuego.NewServer()
+
+		route := fuego.Get(s, "/test", helloWorld,
+			fuego.OptionSecurity(
+				openapi3.SecurityRequirement{
+					"basic":  []string{},
+					"oauth2": []string{"read:users"},
+				},
+			),
+		)
+
+		require.NotNil(t, route.Operation.Security)
+		require.Len(t, *route.Operation.Security, 1)
+		require.Contains(t, (*route.Operation.Security)[0], "basic")
```

Comment:
You could check is empty


```suggestion
		require.Contains(t, (*route.Operation.Security)[0], "basic")
		require.Empty(t, (*route.Operation.Security)[0]["basic"])
```